ociclient/client.go

```diff
@@ -134,6 +138,15 @@ func (c *client) GetManifest(ctx context.Context, ref string) (*ocispecv1.Manife
 		return nil, err
 	}
 
+	if desc.MediaType == DockerV2Schema1MediaType || desc.MediaType == DockerV2Schema1SignedMediaType {
+		c.log.V(3).Info("found v1 manifest -> convert to v2")
```

Comment:
should be verbosity 5 or 7
```suggestion
		c.log.V(7).Info("found v1 manifest -> convert to v2")
```